.github/workflows/publish.yml

```diff
@@ -45,7 +45,14 @@ jobs:
 
       - name: Install HDF5 for macOS arm64
         if: ${{ matrix.os_arch[1] == 'macosx_arm64' }}
-        run: brew install hdf5
+        run: |
+          # pkg-config formula is deprecated but it's still installed
+          # in GitHub Actions runner now. We can remove this once
+          # pkg-config formula is removed from GitHub Actions runner.
+          brew unlink pkg-config@0.29.2
```

Comment:
https://github.com/actions/runner-images/blob/ffd458e26366fb9f0e99222020cbaf8bce86a254/images/macos/macos-15-Readme.md?plain=1#L68

brew apparently needs a specific version to unlink from. This version came from here.